### ISSUE TITLE
Make the color of the MyLocationNewOverlay circle changable

### ISSUE THREAD

**Issue by nycex** [osmdroid/osmdroid#1597]
## Issue Type
- [x] Improvement

## Description and/or steps/code to reproduce the problem
The field `mCirclePaint` of MyLocationNewOverlay is protected and hardcoded to `#6464ff`, it would be nice if the color would be customizable, though.

6.1.6

**Comment by spyhunter99** [comment]
If it's protected, you may be able to sub class it and add a setter.

**Comment by nycex** [comment]
Well yeah, I actually use reflection for it currently 😄, but I think it would be a good idea to have a normal way for it in the osmdroid api.

**Comment by monsieurtanuki** [comment]
From what I read in the code, this is how we use `mCirclePaint` in different situations:
```java
mCirclePaint.setARGB(0, 100, 100, 255);
mCirclePaint.setAntiAlias(true);

mCirclePaint.setAlpha(50);
mCirclePaint.setStyle(Style.FILL);

mCirclePaint.setAlpha(150);
mCirclePaint.setStyle(Style.STROKE);
```

Beyond @spyhunter99 's workaround, I guess a new method called `MyLocationNewOverlay.setDrawAccuracyPaints(Paint firstPaint, Paint secondPaint)` would do the trick, wouldn't it @nycex ?

**Comment by nycex** [comment]
Yeah I think that would be the best way. For my usecase, I wouldn't really need that, since I want the same color for the fill as for the border, but if an API for this is already being created, why not add that ability.

**Comment by monsieurtanuki** [comment]
@nycex Well, thanks for the suggestion. I'll code it within a month - as you already have a workaround I guess it's ok with you.
